src/sbt-test/scoverage/scala3-coverage-excluded-files/build.sbt

```diff
@@ -4,7 +4,7 @@ scalaVersion := "3.4.2"
 
 libraryDependencies += "org.scalameta" %% "munit" % "0.7.29" % Test
 
-coverageExcludedFiles := ".*/two/GoodCoverage"
+coverageExcludedFiles := ".*/two/GoodCoverage;.*/three/GoodCoverage"
```

Comment:
Do the forward slashes need to be escaped like this ... ???
```scala
coverageExcludedFiles := ".*\\/two\\/GoodCoverage;.*\\/three\\/.*"
```

Reply:
Hey, not sure about slashes. I believe it works as is on Linux which is proven by tests. But maybe you would want to add a windows environment for the checks and we can see if tests pass.

And thanks for checking the PR!